packages/app/src/store/aut-store.ts

```diff
@@ -69,6 +78,9 @@ export const useAutStore = defineStore({
     setScriptError (err: ScriptError) {
       this.scriptError = err
     },
+    setScale (scale) {
```

Comment:
I don't think we need this, see https://github.com/cypress-io/cypress/pull/20068

Reply:
Confirming, we'll revisit this today in the PR for https://cypress-io.atlassian.net/browse/UNIFY-951